### PR TITLE
Add parameter to people_detector

### PR DESCRIPTION
**Pull request by iory**

### CODE REVIEW

**Comment by iory** [comment]
If this PR merged, I will rewrite opencv_apps's roswiki!

File: src/nodelet/people_detect_nodelet.cpp [hunk 1]
```diff
@@ -17,7 +17,7 @@
 *   * Neither the name of the Kei Okada nor the names of its
 *     contributors may be used to endorse or promote products derived
 *     from this software without specific prior written permission.
-* 
+*
```
Comment (k-okada) [inline src/nodelet/people_detect_nodelet.cpp]: please remove this diff...

File: src/nodelet/people_detect_nodelet.cpp [hunk 2]
```diff
@@ -89,7 +100,7 @@ class PeopleDetectNodelet : public opencv_apps::Nodelet
   {
     do_work(msg, cam_info->header.frame_id);
   }
-
```
Comment (k-okada) [inline src/nodelet/people_detect_nodelet.cpp:92]: this one too

File: src/nodelet/people_detect_nodelet.cpp [hunk 3]
```diff
@@ -208,7 +219,7 @@ class PeopleDetectNodelet : public opencv_apps::Nodelet
     srv.setCallback(f);
 
     hog_.setSVMDetector(cv::HOGDescriptor::getDefaultPeopleDetector());
-
```
Comment (k-okada) [inline src/nodelet/people_detect_nodelet.cpp:211]: this too

**Comment by iory** [comment]
I remove those

**Comment by k-okada** [comment]
Thank you!!

**Pull request merged**